src/test/java/pages/BookingPage.java

```diff
@@ -4,18 +4,19 @@
 
     /** HomePage **/
 
-    public String searchPlaceXPath = "//*[@id='ss']";
+    public String searchPlaceInput = "//*[@id='ss']";
     public String calendarRange = "//span[@class='sb-date-field__icon sb-date-field__icon-btn bk-svg-wrapper calendar-restructure-sb']";
     public String selectGuests = "//label[@id='xp__guests__toggle']";
-    public String checkPricesXPath = "//button[@data-sb-id='main']";
-    public String selectCurrencyXPath = "//button[@data-modal-aria-label  = 'Выберите валюту']";
-    public String selectLanguageXPath = "//button[@data-modal-aria-label  = 'Выберите язык']";
+    public String checkPricesButton = "//button[@data-sb-id='main']";
+    public String selectCurrency = "//button[@data-modal-aria-label  = 'Выберите валюту']";
+    public String selectLanguage = "//button[@data-modal-aria-label  = 'Выберите язык']";
```

Comment:
it's not selectLanguage it's selecteLanguageButton

Reply:
done